packages/satusehat/src/Adaptor.js

```diff
@@ -166,15 +158,13 @@ export function put(path, data, params = {}, callback = s => s) {
  * You can pass Satusehat an array of objects which contains `op`, `path`, and `value` as the body. You can also pass Satusehat query parameters as an object of key value pairs, which will map to parameters
  * in the URL.
  * @example <caption>Update a resource. Equivalent to PATCH `<baseurl>/Organization/abcde`</caption>
- * patch(
- *   "Organization/abcde",
- *    [{
- * "op": "replace", // Operation - `replace` is the only one used to change a specific property or element
- *  "path": "/language", // Path - The name of property/element of resource to be replaced
- *  "value": "id" // Value- The value to be replaced
- *}]
- *
- * );
+ * patch('Organization/abcde', [
+ * {
+ *  op: 'replace', // Operation - `replace` is the only one used to change a specific property or element
```

Comment:
hmm. I'm not crazy about all the  comments here - it makes the example very busy :thinking: 

Every word in an example should be meaningful. Critical, even, to understanding the example. And I'm just not sure these comments are? Although I  can see why you've done it because you're right, it's not an intuitive example

I think if the caption said "Change a property of a resource" (instead of just "update a resource"), the data structure here would make less sense and you can get away with fewer comments.

Reply:
As a side note - is the `op` in a `patch` ALWAYS `replace`? I just checked the docs and that seems to be true.

In that case we should default it so that users don't need to specify it :) Let's not do that now, I'll raise an issue for it